sys-auth/libfprint/libfprint-1.90.1-r1.ebuild

```diff
@@ -10,23 +10,25 @@ HOMEPAGE="https://cgit.freedesktop.org/libfprint/libfprint/ https://github.com/f
 SRC_URI="https://gitlab.freedesktop.org/libfprint/libfprint/-/archive/v${PV}/${PN}-v${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="LGPL-2.1"
-SLOT="0"
+SLOT="2"
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
-IUSE="examples"
+IUSE="examples +introspection"
 
 RDEPEND="dev-libs/glib:2
-	dev-libs/nss
 	>=dev-libs/libgusb-0.3.1
+	dev-libs/nss
 	virtual/libusb:1=
 	x11-libs/gtk+:3
 	x11-libs/pixman
 	x11-libs/libX11
-	x11-libs/libXv"
+	x11-libs/libXv
+	introspection? ( dev-libs/gobject-introspection:= )
```

Comment:
According to what I see in [libfprint/meson.build](https://gitlab.freedesktop.org/libfprint/libfprint/-/blob/2d10d864d8bc4b952cf412f6e18e44412dba759a/libfprint/meson.build#L320-349), there seems to be no linking against any library provided by `dev-libs/gobject-introspection`.
For generation of `*.gir` and `*.typelib` files, `dev-libs/gobject-introspection` should be in `BDEPEND`.

Reply:
I put it in DEPEND+RDEPEND because that's where basically every GNOME package puts introspection.  There is not much hope of cross-compiling introspection in Gentoo (see for example https://maxice8.github.io/8-cross-the-gir/), so it shouldn't make a difference, but I guess I can move this one.  It's on my TODO list to hack it together at some point (maybe after #18813).